camp_fin/views.py

```diff
@@ -1191,9 +1197,143 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class CandidateDetail(CommitteeDetailBaseView):
+"""
```

Comment:
let's move this stuff to a separate file, that file will also need to have django-extensions license in the header.